py/packages/genkit/src/genkit/blocks/prompt.py

```diff
@@ -679,3 +761,793 @@ async def render_user_prompt(
         )
 
     return Message(role=Role.USER, content=_normalize_prompt_arg(options.prompt))
+
+
+logger = structlog.get_logger(__name__)
```

Comment:
let's move this to the start of the module if youre using it. or we can consider removing debugging logs.